javacutil/src/main/java/org/checkerframework/javacutil/PluginUtil.java

```diff
@@ -34,11 +34,31 @@
     public static final String JAVAC_PATH_OPT = "-javacJar";
 
     /**
-     * Option name for specifying an alternative jdk.jar location. The accompanying value MUST be
-     * the path to the jar file (NOT the path to its encompassing directory)
+     * Option name for specifying an alternative location for the annotated JDK.
+     *
+     * <p>In case of Java 8, an alternative jdk.jar location. The accompanying value MUST be the
+     * path to the jar file (NOT the path to its encompassing directory).
+     *
+     * <p>In case of Java 9+, a directory containing annotated &lt;module&gt;.jar. The supplied
+     * directory MUST contain {@code Patch_Modules_argfile} containing all {@code --patch-module
+     * <jdkModule>=<annotatedModule>}.
      */
-    public static final String JDK_PATH_OPT = "-jdkJar";
+    public static final String JDK_PATH_OPT;
+
+    static {
+        if (PluginUtil.getJreVersion() == 8) {
+            JDK_PATH_OPT = "-jdkJar";
+        } else {
+            JDK_PATH_OPT = "-jdkModulesPath";
+        }
+    }
 
+    /**
+     * Takes list of fileNames and creates list of files.
```

Comment:
`Convert a list of file names into a list of files.`
`@param fileNames list of file names`

Reply:
Done.